lib/ruby_lsp/server.rb

```diff
@@ -301,10 +301,14 @@ def run_initialize(message)
 
       # Not every client supports dynamic registration or file watching
       if @global_state.client_capabilities.supports_watching_files
-        send_message(Request.register_watched_files(@current_request_id, "**/*.rb"))
+        send_message(Request.register_watched_files(@current_request_id, "didChangeWatchedFilesWorkspace", "**/*.rb"))
```

Comment:
The full name seems a little redundant since all will begin `didChangeWatchFiles`, how about `workspace` (or `:workspace`)?

Alternatively, could we auto-generate this ID, e.g. by taking a hash of the pattern argument?